DataFormats/CaloRecHit/interface/CaloRecHit.h

```diff
@@ -4,27 +4,63 @@
 #include "DataFormats/DetId/interface/DetId.h"
 #include <ostream>
 
+#ifdef __CUDA_ARCH__
+  __constant__
+#else
+  constexpr
+#endif
+uint32_t masks[] = {
```

Comment:
this is named to simply to be a global symbol.
Please move inside CaloRecHit  or give this a more creative name.

Reply:
how about anonymous namespace?